misk-redis/src/testFixtures/kotlin/misk/redis/testing/RealRedisTestModule.kt

```diff
@@ -0,0 +1,52 @@
+package misk.redis.testing
+
+import com.google.inject.Provides
+import com.google.inject.util.Modules
+import jakarta.inject.Singleton
+import misk.inject.KAbstractModule
+import misk.redis.JedisPooledWithMetrics
+import misk.redis.RedisClientMetrics
+import misk.redis.RedisModule
+import misk.redis.RedisReplicationGroupConfig
+import misk.testing.parallelTestIndex
+import redis.clients.jedis.ConnectionPoolConfig
+import redis.clients.jedis.UnifiedJedis
+import wisp.deployment.Deployment
+
+/**
+ * Installs a real redis for testing with support for parallel tests.
+ */
+class RealRedisTestModule(
```

Comment:
Adding this separate test module in the testFixtures source dir to avoid the need for changes to the public APIs for RedisModule which is used in production code.
This also allowed me to have more sensible defaults for testing, such as `useSsl = false` and `requiresPassword = false`.